src/common/iop_profile.c

```diff
@@ -124,7 +124,7 @@ static void _transform_from_to_rgb_lab_lcms2(const float *const image_in, float
   if(rgb_profile == NULL)
   {
     rgb_profile = dt_colorspaces_get_profile(DT_COLORSPACE_LIN_REC2020, "", DT_PROFILE_DIRECTION_WORK)->profile;
-    fprintf(stderr, _("unsupported working profile %s has been replaced by Rec2020 RGB!\n"), filename);
+    dt_print(DT_DEBUG_ALWAYS, _("unsupported working profile %s has been replaced by Rec2020 RGB!\n"), filename);
```

Comment:
Remove translation? We don't do it for any other log message...

Reply:
Arrgh. Missed that